### PR TITLE
Fixes/final tweaks

### PR DESCRIPTION
**Pull request by lili2311**
@womenhackfornonprofits/whfnp

### CODE REVIEW

File: src/faq.html [hunk 1]
```diff
@@ -29,27 +29,28 @@ <h2 class="section-heading">Frequently Asked Questions: Volunteers</h2>
         <li>Teaching workshops</li>
         <li>Consultation, advice on expansion, growth strategy, etc</li>
       </ul>
+      If in doubt please feel free to get in touch and have a chat.
```
Comment (elischutze) [inline src/faq.html]: should we make 'get in touch' be a link? or just F it and deploy?

File: src/thanks.html [hunk 2]
```diff
@@ -9,7 +9,7 @@ <h1 class="title-heading">Contact Us</h1>
 </section>
 <section class="col-xs-offset-1 col-xs-10 col-sm-8 col-sm-offset-2 col-md-6 col-md-offset-3 col-lg-3 col-lg-offset-4 col-xl-2 col-xs-offset-5 position central">
 	<h2>Thanks!</h2>
-	<p>Someone will be in touch shortly :) </p>
+	<h4>Your message has been sent and someone will be in touch shortly :)</4>
```
Comment (elischutze) [inline src/thanks.html]: h4!

File: src/faq.html (hunk 1 above)
Comment (lili2311) [reply src/faq.html]: fixed

File: src/thanks.html (hunk 2 above)
Comment (lili2311) [reply src/thanks.html]: fixed

**Comment by VAO11** [comment]
Hi @lili2311, all the fixes look great. The only thing I noticed other than what Eli spotted is the text on the **About Us** page:
image

The text is vertically aligned to the top in IE. It appears as centred in Chrome and Firefox. Other than that this looks good to deploy.

**Comment by herecomesjaycee** [comment]
Tested in Edge, the link headers works the same as Chrome. 

image
image

**Comment by lili2311** [comment]
ty! working on the IE bug :)

**Comment by VAO11** [comment]
Happy to merge and raise separate issue for the IE11 banner issue.

**Pull request merged**